testing/munin/APKBUILD

```diff
@@ -0,0 +1,84 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=munin
+pkgver=2.0.25
+pkgrel=0
+pkgdesc="A distributed monitoring/graphing tool"
+url="http://munin-monitoring.org/"
+arch="noarch"
+license="GPL"
+perl_modules="perl-rrd perl-net-snmp perl-log-log4perl perl-html-template
+	perl-net-ssleay perl-net-server perl-date-manip perl-io-socket-inet6
+	perl-file-copy-recursive perl-fcgi perl-uri"
+depends="$pkgname-node"
+makedepends="perl perl-module-build"
+install="$pkgname.post-install"
+subpackages="$pkgname-node"
+source="http://downloads.munin-monitoring.org/munin/stable/$pkgver/$pkgname-$pkgver.tar.gz
+	munin-config.patch
+	munin.cron.d
+	munin-node.initd
+	munin.logrotate
+	munin-node.logrotate"
+
+_builddir="$srcdir/$pkgname-$pkgver"
+prepare() {
+	local i
+	cd "$_builddir"
+	for i in $source; do
+		case $i in
+			*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1
+		esac
+	done
+}
```

Comment:
This is now a default behaviour when the `prepare()` function is not declared in APKBUILD, so you can just omit it. It’s currently undocumented, I found it yesterday when reading source code of abuild [here](http://git.alpinelinux.org/cgit/abuild/tree/abuild.in#n543).